test/WebJobs.Script.Tests.Integration/WebJobs.Script.Tests.Integration.csproj

```diff
@@ -38,7 +38,7 @@
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="3.1.0" />
     <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.3.0" />
     <PackageReference Include="Microsoft.Azure.EventHubs" Version="2.1.0" />
-    <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="2.0.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="2.0.1" />
```

Comment:
I don't think we actually need this reference here, since this will be transitively referenced with the refs to Script/WebHost. Let's remove it next time, so it reduces the chances of a mismatch. But it's fine for now.